pennylane_qiskit/qiskit_device.py

```diff
@@ -205,6 +205,9 @@ def apply(self, operation, wires, par):
 
         if operation == "QubitStateVector":
 
+            if self.backend_name == "unitary_simulator":
+                raise QuantumFunctionError("The QubitStateVector operation is not supported on the unitary simulator backend.")
+
```

Comment:
These lines were added as per this issue: https://github.com/Qiskit/qiskit-aer/issues/497